components/BurgerItems.tsx

```diff
@@ -40,6 +40,11 @@ const BurgerItems = ({ open, setOpen }) => {
           </h2>
         </a>
       </NextLink>
+      <NextLink href="/faq" as="/faq">
+        <a>
+          <h2 role="term"> {language === ENGLISH ? 'FAQ' : 'FAQ'}</h2>
```

Comment:
@vsauceda-cfa If the value is the same in either language, I think you can just hardcode it instead of doing a boolean check.